src/graph/backend/dnnl/kernels/sdp_decomp_config.cpp

```diff
@@ -478,8 +488,8 @@ impl::status_t sdp_decomp_config_t::record_input_offset(
             mm2 = cur_op;
         }
     }
-    if (impl::utils::one_of(nullptr, mm1, mm2)) return status::invalid_graph;
-
+    VCHECK_SDP_DECOMP(mm1 != nullptr && mm2 != nullptr, status::invalid_graph,
+            "Failed to find matmul1 or matmul2");
```

Comment:
Just curious, in what condition will this happen?

Reply:
for example, matmul + sub + softmax + matmul will result in mm1 == nullptr. And maybe some large partition supported patterns are not supported by decomp kernel. They can be filled out in this function.